src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_SIZE = 100;
+    private static Task[] tasks=new Task[MAX_SIZE];
+    public static Scanner inputScanner= new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        initialiseMike();
+    }
+    public static void initialiseMike(){
+        Task.printLines();
+        System.out.println("Hello! I'm Mike!\nEnter your name:");
+        Task.printLines();
+        String nameOfUser= inputScanner.nextLine();
+        Task.printLines();
+        System.out.println("Alright " + nameOfUser + " , What can I do for you?");
+        Task.printLines();
+        String command=inputScanner.nextLine();
+        readInput(command);
+
+    }
+    public static void readInput(String command) {
+        while (!command.equals("bye")) {
+
+            if (command.equals("list")) {
+                printAllTasks(tasks);
+
+            }else if (command.contains("done")) {
+                command=command.replace("done","").strip();
+                int indexOfTask=Integer.parseInt(command)-1;
+                tasks[indexOfTask].completeTask();
+
+            }else if (command.contains("todo")) {
+                String task=command.replace("todo","");
+                Todo todo=new Todo(task);
+                addToTasks(todo);
+
+            } else if (command.contains("deadline")) {
+                String[] words=command.split("/");
```

Comment:
Plural form name used for a collection of objects. Nice!

Reply:
Thank you!